src/main/java/org/mockito/internal/runners/RunnerFactory.java

```diff
@@ -16,9 +21,34 @@
  */
 public class RunnerFactory {
 
+    /**
+     * Creates silent runner implementation
+     */
     public RunnerImpl create(Class<?> klass) throws InvocationTargetException {
+        return create(klass, new Supplier<MockitoTestListener>() {
+            public MockitoTestListener get() {
+                return new NoOpTestListener();
+            }
+        });
+    }
+
+    /**
+     * Creates strict runner implementation
+     */
+    public RunnerImpl createStrict(Class<?> klass) throws InvocationTargetException {
+        return create(klass, new Supplier<MockitoTestListener>() {
+            public MockitoTestListener get() {
+                return new MismatchReportingTestListener(new ConsoleMockitoLogger());
+            }
+        });
+    }
```

Comment:
Good point. It feels that the current implementation is simpler then when I add conditional logic.